ape_ens/utils/namehash.py

```diff
@@ -0,0 +1,53 @@
+import codecs
+import functools
+
+try:
+    from ens.utils import raw_name_to_hash  # type: ignore
+except ImportError:
+    raw_name_to_hash = None
+from eth_utils import keccak
+from hexbytes import HexBytes
```

Comment:
I think this is required to go after all the normal imports
```suggestion
from eth_utils import keccak
from hexbytes import HexBytes

try:
    from ens.utils import raw_name_to_hash  # type: ignore
except ImportError:
    raw_name_to_hash = None
```